lib/client.js

```diff
@@ -98,6 +98,16 @@ const warmupLimit = 32;
  * @property {Number} protocolOptions.maxVersion When set, it limits the maximum protocol version used to connect to
  * the nodes.
  * Useful for using the driver against a cluster that contains nodes with different major/minor versions of Cassandra.
+ * @property {Boolean} protocolOptions.noCompact When set to true, enables the NO_COMPACT startup option.
+ * <p>
+ * When this option is supplied <code>SELECT</code>, <code>UPDATE</code>, <code>DELETE</code>, and <code>BATCH</code>
+ * statements on <code>COMPACT STORAGE</code> tables function in "compatibility" mode which allows seeing these tables
+ * as if they were "regular" CQL tables.
+ * </p>
+ * <p>
+ * This option only effects interactions with interactions with tables using <code>COMPACT STORAGE</code> and is only
+ * support by C* 4.0+ and DSE 6.0+.
```

Comment:
Actually, the option was added in 3.0.16 & 3.11.2 as well, so will update this comment.

Reply:
Missing change :)

Reply:
D'oh!  I'll fix that :)